File: 03_c/day01_swap_numbers.c
#include <stdio.h>

int main() {
    int a = 5;
    int b = 10;
    int temp = 0;

    temp = b;
    b = a;
    a = temp;

    printf("Value of a : %d\n", a);
    printf("Value of b : %d\n", b);
}

File: 05_csharp/Day01_Swap_Numbers.cs
public class Day01_Swap_Numbers
{
    public static void Run()
    {
        int a = 5;
        int b = 10;

        int temp = a;
        a = b;
        b = temp;

        Console.WriteLine($"a = {a}");
        Console.WriteLine($"b = {b}");
    }
}